processing/src/main/java/io/druid/query/groupby/GroupByQueryQueryToolChest.java

```diff
@@ -474,10 +477,15 @@ public Object apply(Row input)
           {
             if (input instanceof MapBasedRow) {
               final MapBasedRow row = (MapBasedRow) input;
-              final List<Object> retVal = Lists.newArrayListWithCapacity(2);
+              final List<Object> retVal = Lists.newArrayListWithCapacity(1 + dims.size() + aggs.size());
               retVal.add(row.getTimestamp().getMillis());
-              retVal.add(row.getEvent());
-
+              Map<String, Object> event = row.getEvent();
+              for (DimensionSpec dim : dims) {
+                retVal.add(event.get(dim.getDimension()));
```

Comment:
hopefully it's dimension name, since the cache key only includes the original name, not the aliased one